packages/wpcom-checkout/src/payment-methods/ideal.tsx

```diff
@@ -139,11 +140,15 @@ function BankSelector( {
 } ) {
 	const { __ } = useI18n();
 	const bankOptions = getBankOptions( __ );
+	const classes = classNames( 'form-select', {
```

Comment:
I was trying to figure out where these styles come from. It appears to be `client/components/forms/form-select/style.scss`. The downside of using that style to fix this `<select>` is that it's in a different package; if someone tried to use this component from `@automattic/wpcom-checkout` outside of calypso (or any page where that css file wasn't already included), then they wouldn't have that style available.

It may be better to do one of three things; either:

1. Include whatever styles are needed directly in this file using the CSS-in-JS approach used in the rest of the file. The downside is that you may need a lot of styling (I'm not sure actually which rules are required to make this look nice and which are extraneous).
2. Import and use a different component for the `<select>` that already has styling. This package doesn't have any component libraries directly included, which is unfortunate. You could create a custom one like we've done with [Field](https://github.com/Automattic/wp-calypso/blob/0d203c15168af5266cc036f284cba74211a4dbf0/packages/wpcom-checkout/src/field.tsx#L8), possibly duplicating [FormSelect](https://github.com/Automattic/wp-calypso/blob/0d203c15168af5266cc036f284cba74211a4dbf0/client/components/forms/form-select/index.jsx#L6). The downside is that you'll probably be duplicating code that already exists, increasing the package size and therefore the bundle size of anywhere that uses it.
3. Import a styled `<select>` element from some other package; it looks like `@automattic/components` doesn't have one for some reason. You could probably use [SelectControl](https://wordpress.github.io/gutenberg/?path=/story/components-selectcontrol--default) from `@wordpress/components`. If you take this path, you'll need to add that package as a dependency to [this package](https://github.com/Automattic/wp-calypso/blob/0d203c15168af5266cc036f284cba74211a4dbf0/packages/wpcom-checkout/package.json#L39). The downside is that new imports can greatly increase bundle size, which is fine if they're going to be helpful in many places, but not great if this is the only place that package will be used.

I'd suggest starting with option 1 to see if there's a minimal set of styles you can add that will fix this element, and if not, then look into the other choices.

Reply:
Went with option 1, it was quite a bit of CSS code, but it wasn't too problematic to implement. Through testing I believe I included only the CSS required to match the styling which inevitably was most (if not all) of the styling from `.form-select`.